server/clearbit.js

```diff
@@ -387,11 +386,26 @@ export default class Clearbit {
     });
   }
 
-  fetchProspects(query, company_traits = {}) {
-    return this.client.prospect({ ...query, email: true }).then((prospects) => {
-      this.hull.logger.info("clearbit.prospector.success", { action: "prospector", message: `Found ${prospects.length} new Prospects`, company_traits, prospects });
-      prospects.map(this.saveProspect.bind(this, company_traits));
-      return prospects;
+  fetchProspects(query = {}, company_traits = {}) {
+    const { titles = [], domain, role, seniority, limit = 5 } = query;
+    if (titles.length === 0) {
```

Comment:
Why we are pushing null element to this `titles` array ? I'm testing it now and it may produce some unwanted scenario. After that operation our `title` array will look like this: `[ null ]` so Promise.mapSeries will run and then we will pass `null` as `title` to `params`. I think we want to avoid that.